docs/bot-usage.md

```diff
@@ -13,7 +13,82 @@ For example, to check if the bot is alive you can write this comment:
 
 And the bot will reply to you.
 
-## Creating experiments
+## Table of Contents
+
+[h-toc]: #table-of-contents
+
+* [Tutorial: creating an experiment for a PR][h-tutorial]
+* [Available experiment modes][h-experiment-modes]
+* [Automatic experiment names][h-experiment-names]
+* Commands reference:
+  * [Creating experiments][h-cmd-run]
+  * [Editing experiments][h-cmd-edit]
+  * [Aborting experiments][h-cmd-abort]
+* Troubleshooting:
+  * [Regenerating a report if it failed][h-troubleshooting-retry-report]
+  * [Reload the list of GitHub team members][h-troubleshooting-reload-teams]
+
+## Tutorial: creating an experiment for a PR
+
+[h-tutorial]: #tutorial-creating-an-experiment-for-a-pr
+
+When you want to create an experiment to check if a PR is breaking some
+third-party code, the first thing is to request a **try build** to get a valid
+toolchain. You can do that with the `@bors try` GitHub comment.
+
+Once the try build is done, you can get the SHA of the start and end commits.
+The end commit is the commit by bors used for the try build, and the start
+commit is the first parent of that (another merge commit by bors). You must
+prefix the start commit with `master#`, and the end commit with `try#`.
+
+Then, you choose the [experiment mode you want to use][h-experiment-modes], and
+type up the command in your GitHub PR:
+
+```
+@craterbot run start=master#fullhash end=try#fullhash mode=check-only
```

Comment:
Short hashes won't work since try builds are stored with the full hash as the name, and Crater needs to fetch them. This can be fixed by resolving the hashes on the Crater side, but the code for that needs to be written.